app/code/Magento/InventoryCatalog/view/adminhtml/ui_component/product_form.xml

```diff
@@ -171,4 +171,25 @@
             </insertListing>
         </modal>
     </fieldset>
+
```

Comment:
Remove redundant line break.

Reply:
Ok. I'll remove all redundant line breaks.

Reply:
I'll push everything by force push into this commit.